test/system/events_test.rb

```diff
@@ -356,4 +356,11 @@ class EventsTest < ApplicationSystemTestCase
     end
     assert_text 'Watch中'
   end
+
+  test 'show user full_name next to user login_name' do
+    event = events(:event2)
+    login_user 'kimura', 'testtest'
+    visit event_path(event)
```

Comment:
```suggestion
    login_user 'kimura', 'testtest'
    visit event_path(events(:event2))
```

一箇所でしか使ってなかったらこれでいいかもです〜

Reply:
確かにそうですね！
「Commit suggestion」ボタンでCommitしたんですが、CIが通っていれば問題ないですかね……？